src/process.cpp

```diff
@@ -625,6 +664,8 @@ namespace proc {
 
         ctx.elevated = elevated.value_or(false);
         ctx.auto_detach = auto_detach.value_or(true);
+        ctx.wait_all = wait_all.value_or(true);
+        ctx.exit_timeout = std::chrono::seconds { exit_timeout.value_or(5) };
```

Comment:
This section sets the value to our default for apps which haven't been edited yet?

Reply:
Yep, this is basically the equivalent of our `config_t` defaults in config.cpp.